src/snowflake/snowpark/_internal/analyzer/snowflake_plan.py

```diff
@@ -563,7 +563,9 @@ def large_local_relation_plan(
             is_generated=True,
         )
         insert_stmt = batch_insert_into_statement(
```

Comment:
Is this function ever called anywhere else? If so maybe make the paramstyle have a default value that matches the old value?

Reply:
Its only called once here